res/TensorFlowLiteRecipes/Net_Gather_SparseToDense_AddV2_000/test.recipe

```diff
@@ -21,20 +21,20 @@ operand {
   shape { dim: 1 dim: 1 }
   filler {
       tag: "explicit"
-      arg: "3" arg: "5"
+      arg: "0"
   }
 }
 operand {
   name: "shape_sparse2dense"
   type: INT64
   shape { dim: 1 }
-  filler { tag: "explicit" arg: "1" arg: "2" }
+  filler { tag: "explicit" arg: "3" }
```

Comment:
`shape_sparse2dense` operand means output_shape of `sparse_to_dense`. 
Since `ofm_sparse2dense.shape` is 3, this value must be 3.